diff -up distlib-0.3.4/distlib/compat.py.1~ distlib-0.3.4/distlib/compat.py
diff -up distlib-0.3.4/setup.py.1~ distlib-0.3.4/setup.py
--- distlib-0.3.4/setup.py.1~	2022-06-14 22:12:59.068628086 +0200
+++ distlib-0.3.4/setup.py	2022-06-14 22:13:38.237785333 +0200
@@ -61,10 +61,6 @@ distutils.core.setup(
     packages=[
         'distlib'
     ],
-    package_data={
-        'distlib': ['t32.exe', 't64.exe', 'w32.exe', 'w64.exe',
-                    't64-arm.exe', 'w64-arm.exe'],
-    },
     cmdclass={
         'test': TestCommand,
     },
